packages/build-essentials/src/webpack.config.js

```diff
@@ -33,8 +33,8 @@ const webpackConfig = {
                 loader: 'json-loader'
             },
             {
-                test: /\.(woff|woff2|eot|svg)$/,
-                loader: 'url-loader?limit=100000'
+                test: /\.(woff|woff2)$/,
```

Comment:
In that case you shouldn't remove the loader from the dependencies, right? https://github.com/neos/neos-ui/pull/821/files/b0242e333c0d12bf7fbeb031e3c6fe10a2831143#diff-6e183e897d30da3e86a99a9c16857780L52